src/css/styles.scss

```diff
@@ -249,24 +249,24 @@ main {
   margin: 20px;
   padding: 15px;
   width: 250px;
+}
 
-  .recipe-card:hover .card-photo-preview {
-    opacity: 50%;
-    transition: .8s ease;
-  }
+.recipe-card:hover .card-photo-preview {
```

Comment:
Loved the nesting, but it looks like the original css was a bit too complex to begin with for nesting here. I think un-nesting was the right call